pkg/core/interop_system.go

```diff
@@ -216,19 +216,23 @@ func runtimeGetTrigger(ic *interop.Context, v *vm.VM) error {
 // runtimeNotify should pass stack item to the notify plugin to handle it, but
 // in neo-go the only meaningful thing to do here is to log.
 func runtimeNotify(ic *interop.Context, v *vm.VM) error {
-	// It can be just about anything.
-	e := v.Estack().Pop()
-	item := e.Item()
+	name := v.Estack().Pop().Bytes()
+	elem := v.Estack().Pop()
+	args := elem.Array()
```

Comment:
Why do you force it being an array? It can be anything.

Reply:
Not anymore according to https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/ApplicationEngine.Runtime.cs#L150